bench/demo.fpcore

```diff
@@ -27,7 +27,7 @@
 (FPCore (x)
  :name "sqrt sqr"
  :alt 
- (! :description (platform libm) (if (< x 0) 2 0))
+ (! :description (platform default) (if (< x 0) 2 0))
```

Comment:
I still don't understand why we're using `:description` here. Why not `:herbie-platform default`? That would have three benefits:
- Descriptions should be strings, not lists
- Plus we wouldn't have to parse out the "platform" if that was the only part of it
- Plus people would be able to use descriptions if they wanted.

Reply:
Also, why is `(platform default)` not the default? Like, why do we have to say it at all?

Reply:
To answer all the questions:

1. We don't have to say `(platform default)` as the default anymore. Even if it is not there, the new case where we use `with-handlers` to catch an exception will handle that. 

2. :description was used as it was the closest thing in the FPBench language that seemed like a good name to store platform names. But I guess I never thought we could make a custom key like `herbie-platform` and that does solve all problems.

3. So like you mentioned, all benefits are taken care of and `:description` can now be used tot store any or all metadata.